bootstrap-salt.sh

```diff
@@ -4457,6 +4469,9 @@ install_opensuse_stable_deps() {
     # shellcheck disable=SC2086
     __zypper_install ${__PACKAGES} || return 1
 
+    # Fix for OpenSUSE 13.2 and 2015.8 - gcc should not be required. Work around until package is fixed by SuSE
+    _EXTRA_PACKAGES="${_EXTRA_PACKAGES} gcc python-devel libgit2-devel" 
```

Comment:
Why do we need these? Specially pygit2 which is not a hard dependency for Salt.

What type of installation triggered this? Stable packages? Git based installation?